docs/security-considerations.rst

```diff
@@ -200,7 +200,7 @@ Now someone tricks you into sending ether to the address of this attack wallet:
 
 ::
 
-    pragma solidity ^0.4.11;
+   pragma solidity ^0.4.23;
```

Comment:
Why does this require 0.4.23 while the others only 0.4.22?

Reply:
With 0.4.22 it throws an error:
`DeclarationError: Function with same name and arguments defined twice.`

Reply:
Oh right, 0.4.22 had the bug. Why not just require 0.4.23 for the all the updated examples with `constructor` so that users won't run into the bug if they change the example?